cmd/set_config.go

```diff
@@ -15,79 +15,75 @@
 package cmd
 
 import (
-    "fmt"
-    "log"
-    "os/user"
-    "path/filepath"
```

Comment:
looks like this file has changes in tab space, please check this once.

Reply:
done